index.html

```diff
@@ -10,6 +10,10 @@
   <link rel="stylesheet" href="css/semantic.min.css" type="text/css">
   <link rel="stylesheet" href="css/style.css" type="text/css" media="screen">
   <link rel="icon" type="image/x-icon" href="images/favicon.png" />
+  <script>
+if (window.location.status === 404) {
+      window.location.href = "/404.html";
+    }</script>
```

Comment:
Google Cloud will take care of showing the 404 page when needed. Please remove this.